### PR TITLE
[release/8.0.1xx-preview6] Update aspnet packages

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change

Bump .net7 stable version
Try to update with the packages that are going to ship.
The aspnet packages were updated using `darc`.

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -4,21 +4,25 @@
...
     </Dependency>
-    <Dependency Name="System.Text.Encodings.Web" Version="8.0.0-preview.6.23279.6" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="System.Text.Encodings.Web" Version="8.0.0-preview.6.23323.4" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha></Sha>
+      <Sha>
+      </Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0-preview.6.23279.6" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0-preview.6.23323.4s" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
```
Comment (jonathanpeppers) [inline eng/Version.Details.xml:22]: The letter `s` doesn't seem right. Were these number changed manually?

**Review by jonathanpeppers — COMMENTED** [review]
What `darc` command did you run to update these? I usually find it helpful to put it in the commit message & PR description.

If it wasn't a `darc` command, we should be using it to avoid typos.

File: eng/Version.Details.xml (hunk 1 above)
Comment (rmarinho) [reply eng/Version.Details.xml:22]: yap they were.

Comment (jonathanpeppers) [reply eng/Version.Details.xml]: Where did these numbers come from? Is there a reason `darc` can't update them for us?

**Review by mattleibow — APPROVED** [review]

**Pull request merged**